src/MutableMap/MutableMap.dfy

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT
+*******************************************************************************/
+
+// RUN: %dafny /compile:0 "%s"
+
+module {:extern "MutableMap"} {:options "/functionSyntax:4"} MutableMap {
```

Comment:
I'd at least move this under the "DafnyLibraries" namespace as the FileIO code is. "MutableMap" has an outside chance of conflicting with other Java code.